clusterloader2/testing/l4ilb/config.yaml

```diff
@@ -117,6 +120,31 @@ steps:
     Method: WaitForControlledPodsRunning
     Params:
       action: gather
+{{if $EXEC_COMMAND}}
+- name: Starting measurements for nodepool upgrade
+  measurements:
+  - Identifier: ServiceCreationLatencyNodepoolUpgrade
+    Method: ServiceCreationLatency
+    Params:
+      action: start
+      waitTimeout: {{$ilbWaitTimeout}}
+      labelSelector: size = {{$LARGE_BACKEND_LABEL}}
```

Comment:
We only verify time taken for the large backend ILBs to recover?

Reply:
I added the measurements for medium and small too.